### PR TITLE
do not report packages maintained by us but having their own repository

### PR DESCRIPTION
**Pull request by FrankMittelbach**
as requested by Ulrike ...

### CODE REVIEW

File: latexbug.dtx [hunk 1]
```diff
@@ -158,6 +155,43 @@
...
+% (latexbug)                ==============
+% (latexbug)                hyperref.sty  ->  Report at: 
+% (latexbug)                    https://github.com/latex3/hyperref/issues 
+% (latexbug)                ==============
+% (latexbug)                
+% (latexbug)                Please note that these files have their own
+% (latexbug)                bug tracker and your issue should therefore
+% (latexbug)                be reported in the indicated repository.
+% (latexbug)                
+% (latexbug)                If the are not necessary to exhibit the problem,
```
Comment (u-fischer) [inline latexbug.dtx]: the -> they

File: latexbug.dtx [hunk 2]
```diff
@@ -806,7 +850,7 @@ hluatex.def  = ignore ,
 hpdftex.def  = ignore ,
 hxetex.def   = ignore ,               
 hyperref.cfg = ignore ,
-hyperref.sty = LaTeX Team
+hyperref.sty = us*
```
Comment (u-fischer) [inline latexbug.dtx:857]: +pdfmanagement-testphase = us* ??

File: latexbug.dtx (hunk 1 above)
Comment (FrankMittelbach) [reply latexbug.dtx]: anything else?

File: latexbug.dtx (hunk 2 above)
Comment (FrankMittelbach) [reply latexbug.dtx:857]: yes, and probably also tagpdf (and perhaps that should move under latex3/...)

Comment (u-fischer) [reply latexbug.dtx:857]: and xcolor.

**Comment by u-fischer** [comment]
the combination of pdfmanagement and hyperref will still error, but I think that can't be avoided currently as it loads bookmark and that is still a third party package. I would have to properly move the code first. So I think it is now fine. The messages are much better ;-)

**Comment by FrankMittelbach** [comment]
> the combination of pdfmanagement and hyperref will still error, but I think that can't be avoided currently as it loads bookmark and that is still a third party package. I would have to properly move the code first. So I think it is now fine. The messages are much better ;-)

as long as bookmark is external to the `latex3/...` I don't really want to call it `us` or `us*` as we can't move issues from one repository to another if it is not within the same "org" (at least to my understanding).  On the other hand, it is probably ok if we push book mark issues into the hyperref repo since eventually it will end up residing there. So perhaps I should make it
`us*`after all and point to an appropriate repository ... what is the cannonical one for it?

**Pull request merged**